src/app/helpers/ssoHelper.js

```diff
@@ -6,7 +6,7 @@ const uuid = require('uuid/v1')
 const dateFormat = require('dateformat')
 const kafkaService = require('../helpers/kafkaHelperService');
 let ssoWhiteListChannels;
-
+const privateBaseUrl = '/private/user/'
```

Comment:
This should be 
const privateBaseUrl = '/private/user'

Dont append "/" at the end. It will be good practice. So that at any place in the code, 
privateBaseUrl  + "/{{ api }}"